package.json

```diff
@@ -280,6 +280,11 @@
             "verbose"
           ]
         },
+        "puppet.editorService.modulepath": {
```

Comment:
Nit: Technically this should be `modulePath` to align with the other config options case convention but since this doesn't result in compile time errors I'm good for it

Reply:
Fixed.